test/modules/standard/Random/bradc/testGetNth.chpl

```diff
@@ -13,7 +16,8 @@ for i in 1..n {
   if (r1 != r2 || r2 != r3) {
     writeln("mismatch at #", i, ": ", r1, " ?= ", r2, " ?= ", r3);
   } else {
-    writeln("#", i, " = ", r2);
+    // Don't print the #s in this test; just check for a match
+    //writeln("#", i, " = ", r2);
```

Comment:
Nope, that's OK.  I wasn't seeing the big picture well enough to understand that there was another test locking in the specific values being generated by the NPB generator.  Too large a patch to review in not enough time.